lib/ash/policy/info.ex

```diff
@@ -146,11 +146,17 @@ defmodule Ash.Policy.Info do
   See the documentation of `can/4` for more.
   """
   @type can_option? :: {:api, module} | {:maybe_is, boolean()}
-  @spec can?(Ash.Resource.t(), atom(), map() | nil, list(can_option?())) :: boolean()
-  def can?(resource, action_or_action_name, actor, opts \\ []) do
+
+  @spec can?(
+          Ash.Resource.t(),
```

Comment:
This should probably also accept a changeset in the same way.

Reply:
Makes sense, will add that to the mix later.

Just to make sure I grep, as in `can(resource, changeset, actor, opts)`?

Reply:
Yep!